### PR TITLE
[1LP][RFR] Fix group view with expression editor, tests fixes

### PR DESCRIPTION
**Pull request by rlbabyuk**
Purpose or Intent
=================
{{pytest: cfme/tests/cloud_infra_common/test_tag_visibility.py}}
expression form fixed in https://github.com/ManageIQ/integration_tests/pull/7435

### CODE REVIEW

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/infrastructure/virtual_machines.py [hunk 1]
```diff
@@ -1307,7 +1307,17 @@ def resetter(self, *args, **kwargs):
...
+    def prerequisite(self):
+        view = None
+        try:
+            view = navigate_to(self.obj, 'VMsOnly')
+        except NavigationDestinationNotFound:
+            view = navigate_to(self.obj.parent, 'VMsOnly')
+        finally:
+            if view is None:
+                raise NavigationDestinationNotFound('No view set during prerequisite')
+        return view
```
Comment (quarckster) [inline cfme/infrastructure/virtual_machines.py]: Why is it required?

File: cfme/configure/access_control/__init__.py [hunk 2]
```diff
@@ -474,7 +485,13 @@ class GroupForm(ConfigurationView):
     cancel_button = Button('Cancel')
     retrieve_button = Button('Retrieve')
 
-    my_company_tags = View.nested(MyCompanyTags)
+    @View.nested
+    class my_company_tags(Tab):  # noqa
+        """ Represents 'My company tags' tab in Group Form """
+        TAB_NAME = "My Company Tags"
+        form = VersionPick({Version.lowest(): View.nested(MyCompanyTagsTree),
+                            '5.9': View.nested(MyCompanyTagsWithExpression)})
```
[the remark this answers is not in the document]
Comment (rlbabyuk) [reply cfme/configure/access_control/__init__.py]: true, removed

**Comment by rlbabyuk** [comment]
PRT fails 3 cases, but couldn't reproduce these failures locally. Will take care of these separately

**Pull request merged**